### DISCUSSION TITLE
最受不了那些中國用語

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#1064] [General]
https://news.ltn.com.tw/news/life/breakingnews/4178181

其實我有些用語在台灣也蠻常被使用的吧，像給力、很火、牛逼、合同等等，不過的確有些真的聽了蠻怪的，也覺得有些詞特別意外在中國是這樣講，像是無尾熊竟然叫"考拉"(koala)，不過文化上的差異本就會造成用詞上的不同，而我們應該給予尊重。

**Comment by Valentinaa0214** [comment]
以前很不能接受中國用語，
但隨著接觸越來越多大陸文化，
及身邊的人也慢慢將中國用語帶入生活中，
漸漸也就習慣了，有時候甚至覺得他們的用詞還挺有意思的。

**Comment by Pietra1226** [comment]
我覺得他們電影翻譯太直白了，少了一點趣味或是韻味，但相反的他們能更直接體現該電影的重點跟該電影的主題，有好有壞啦。
但我還是喜歡我們的翻譯，聽起來很吸引人

**Comment by Doreen331** [comment]
比起中國用語，我個人比較討厭支語警察，由有時候只是說了一個詞，甚至也不知道是大陸用語，就要馬上被糾正，或是有時寫字為了方便快速，寫了一兩個簡體字，還要被要求重寫，真的duck不必😍

**Reply by lyien1021** [reply]
真的！講什麼用語還要被管？最好大家都不要講外來語

**Comment by Twenty0522** [comment]
中國用語有些簡單粗暴，在某些場合使用會覺得措詞到位，但若是正當的場合可以多加使用委婉文言的用詞。
在日常生活中遇到會以尊重的態度面對，對事而不對政治。

**Comment by ghost** [comment]
我覺得中國用語在娛樂方面還挺喜感，可能也是受了一些影片創作者的影響吧，以前每每想到中國說的大外宣，都不免會擔憂是否我只是被洗腦了，現在倒是看開了，不要什麼是都和政治掛勾，畢竟那不是我能改變的部分，我只要輕鬆度過每一天就足以

**Comment by Sylvie0601** [comment]
我其實在日常也超級常用"牛逼"這個詞的ㄟ，感覺中國用語真的蠻常見的，但我覺得有時候感覺還是要注意措辭，不然有時候聽起來可能會覺得太粗魯，聽的人覺得不太舒服之類的。

**Comment by yuxinlin93** [comment]
其實要用什麼詞自己高興就好了，但中國用語有時候會比較直接，聽的人可能也會不太高興，所以平常都會避免用中國用語，而且翻譯配音什麼的還是聽台灣用詞會比較親切

**Comment by harrylee1971** [comment]
有點簡單但是又粗劣，還是繁體字的內容比較有文學意味一點

**Comment by a45320614** [comment]
我的看法可能有點逆風，我聽到朋友講這些中國用語的時候都會提醒他們，也就是俗稱的「支語警察」，但我也不會強迫朋友不能說中國用語不然就絕交之類的。我也曾經跟朋友討論過這個議題，有些人覺得沒甚麼大不了的、不要大驚小怪、美國用語不是也常常被使用，有些人則會盡量不使用到中國用語但不小心用了也不會怎麼樣，正反兩方的論點都了解及討論過後我還是堅持自己的立場及作法。每個人的想法或許都有些不同，但尊重及包容這些聲音才會使自己成長。

**Comment by HsuChen88** [comment]
我自己說中國用語時多半是調侃某個梗的時候，但如果被糾正還真的會覺得對方很沒幽默感。

**Comment by BlahFish1006** [comment]
我覺得語言就是要拿來用，像我們話間可能也會用到來自英日韓的用語，就相對比較不會被放大檢視，
個人認為沒必要為反中而反中，自己用起來喜歡、使用合乎時宜，若覺得在這個時機點適合用這樣的遣詞來表達更清楚，
那又有什麼不行呢？

**Comment by wenchi9023** [comment]
一開始我不喜歡中國用語，覺得很做作且無聊，但有一部分可能是不習慣吧!
但後來覺得生活中發生的事情用中國用語來表示反而更到味，挺有意思的哈哈

**Comment by lance111l** [comment]
语言也需要创新发展  可以理解

**Comment by Lipunpun** [comment]
真的牛逼，咱大中國的用語肯定是yyds! 到四川就是要吃火鍋，夏天吃火鍋，給力。

**Comment by zilinng** [comment]
我覺得語言本來就只是用來溝通的工具，只要能達到表達自己想法的目的，用中國用語、台灣用語沒什麼差吧！當然大家都有不喜歡的權利，但真的沒必要去貶低或是辱罵！

**Comment by tonyYZL** [comment]
雖然有時候聽到別人講中國用語會突然覺得怪怪的，像是「影片」講成「視頻」，不過聽得懂就好了，不會刻意去糾正。我覺得有些人對中國用語可能有點反應過度，其實這沒什麼，就單純說話方式的差異罷了。
題外話，我個人比較不能理解寫簡體字跟用簡體發文的人（這裡的寫簡體字是指完全使用簡體的那種，開玩笑的例外）

**Comment by curtis1017** [comment]
我看到有一個報導統整2021十大網路用語流行的先後出序，依序是「覺醒年代」「YYDS」「雙減」「破防」「元宇宙」「絕絕子」「躺平」「傷害性不高，侮辱性極強」「我看不懂，但我大受震撼」和「強國有我」，我覺得這些網路流行用語有些能隨著科技的發展而出現，例如「元宇宙」這個詞的定義為元宇宙（英語：metaverse），或稱為後設宇宙、形上宇宙、元界、魅他域、超感空間、虛空間，是一個聚焦於社交連結的3D[虛擬世界]之網路。元宇宙主要探討一個[持久化]和[去中心化的線上[三維]虛擬環境。

**Comment by Lipunpun** [comment]
我是覺得不會到不能接受啦，有的時候倒是覺得蠻好笑的，像是yyds 變成是一種趨勢，讓年輕人講話更有連結，促進社會的發展，從這短短的幾個字，讓台灣的GDP上升一個百分點，我認為是一件很好的文化，背後帶起的產業、行銷手法我真的不敢想像。

**Comment by pigeon9975** [comment]
有人可能會覺得中國用語比較粗俗，講起來不好聽之類的，但其實有很多我們的話講出來也不是那麼好聽啊，像是甚麼ㄎ或是ㄍ開頭的字之類的，但我感覺在日常生活中隨口講出這些話的時候卻不會受到那麼大的反彈...雖然我也不太喜歡那些中國用語，但還是給予尊重啦，在合適的場合講合適的話就好了！這個部分...就看成一種文化融合的表現吧！
